Doc/library/functions.rst

```diff
@@ -1545,6 +1556,9 @@ are always available.  They are listed here in alphabetical order.
    Rather than being a function, :class:`tuple` is actually an immutable
    sequence type, as documented in :ref:`typesseq-tuple` and :ref:`typesseq`.
 
+   .. versionchanged:: 3.7
+      Keyword arguments are no longer accepted.
```

Comment:
`tuple()` has one parameter `iterable`. Perhaps something like "\*iterable\* is now a positional-only parameter." would be more accurate?

Reply:
+1. Also for other functions.

Reply:
@zhangyangyu So for int and friends, we should have `*x* is now a positional-only parameter`?

Reply:
Or "The first parameter is now positional-only."